pkgs/development/libraries/x264/default.nix

```
@@ -1,40 +1,29 @@
 { stdenv
 , lib
 , fetchFromGitLab
-, fetchpatch
 , nasm
 , enableShared ? !stdenv.hostPlatform.isStatic
 }:
 
 stdenv.mkDerivation rec {
   pname = "x264";
-  version = "unstable-2021-06-13";
+  version = "unstable-2023-10-01";
```

```suggestion
  version = "0-unstable-2023-10-01";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

---

Maybe like this?
```patch
+version = "0.164-unstable-2023-10-01"; # 164 is ABI version
```

---

Probably not, since `0.164` is not a release version.